src/vs/workbench/contrib/quickaccess/browser/viewQuickAccess.ts

```diff
@@ -104,7 +104,7 @@ export class ViewQuickAccessProvider extends PickerQuickAccessProvider<IViewQuic
 			for (const view of viewContainerModel.allViewDescriptors) {
 				if (this.contextKeyService.contextMatchesRules(view.when)) {
 					result.push({
-						label: view.name,
+						label: view.name.original,
```

Comment:
Use original here to display original text when other places are localized

Reply:
Not sure I quite understand what you mean here. Why original? (also could you add a comment saying why)